src/app/content/highlights/components/utils/onClickOutside.ts

```diff
@@ -46,6 +46,7 @@ export const useOnClickOutside = (
   cb: (e: MouseEvent) => void,
   eventOptions?: EventListenerOptions
 ) => {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
its necessary to put this just because `onClickOutside` is a separate function? or is it because of eventOptions?

Reply:
Yes, it is because of `onClickOutside`
This error is showing up for all useEffects that are using this structure:
```
React Hook useEffect received a function whose dependencies are unknown. Pass an inline function instead.
```